fast/stages/02-networking-nva/variables.tf

```diff
@@ -0,0 +1,253 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "billing_account_id" {
+  # tfdoc:variable:source 00-bootstrap
+  description = "Billing account id."
+  type        = string
+}
+
+variable "custom_adv" {
+  description = "Custom advertisement definitions in name => range format."
+  type        = map(string)
+  default = {
+    cloud_dns                 = "35.199.192.0/19"
+    gcp_all                   = "10.128.0.0/16"
+    gcp_dev_ew1               = "10.128.128.0/19"
+    gcp_dev_ew4               = "10.128.160.0/19"
+    gcp_landing_trusted_ew1   = "10.128.64.0/19"
+    gcp_landing_trusted_ew4   = "10.128.96.0/19"
+    gcp_landing_untrusted_ew1 = "10.128.0.0/19"
+    gcp_landing_untrusted_ew4 = "10.128.32.0/19"
+    gcp_prod_ew1              = "10.128.192.0/19"
+    gcp_prod_ew4              = "10.128.224.0/19"
+    googleapis_private        = "199.36.153.8/30"
+    googleapis_restricted     = "199.36.153.4/30"
+    rfc_1918_10               = "10.0.0.0/8"
+    rfc_1918_172              = "172.16.0.0/12"
+    rfc_1918_192              = "192.168.0.0/16"
+  }
+}
+
+variable "data_dir" {
+  description = "Relative path for the folder storing configuration data for network resources."
+  type        = string
+  default     = "data"
+}
+
+variable "dns" {
+  description = "Onprem DNS resolvers"
+  type        = map(list(string))
+  default = {
+    onprem = ["10.0.200.3"]
+  }
+}
+
+variable "folder_id" {
+  # tfdoc:variable:source 01-resman
+  description = "Folder to be used for the networking resources in folders/nnnnnnnnnnn format. If null, folder will be created."
+  type        = string
+  default     = null
+  validation {
+    condition = (
+      var.folder_id == null ||
+      can(regex("folders/[0-9]{8,}", var.folder_id))
+    )
+    error_message = "Invalid folder_id. Should be in 'folders/nnnnnnnnnnn' format."
+  }
+}
+
+variable "gke" {
```

Comment:
I think this is never used. 
Should also be removed from 02-networking.

I would also rename 02-networking to 02-networking-hs-vpn (or something similar)

Reply:
done and sent a [separate PR](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/pull/479) for 02-networking